dev-packages/browser-integration-tests/suites/integrations/lazyLoad/moduleMetadataIntegration/init.js

```diff
@@ -0,0 +1,10 @@
+import * as Sentry from '@sentry/browser';
+
+Sentry.init({
+  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  integrations: [],
+});
+
+window.Sentry = {
+  ...Sentry,
```

Comment:
```suggestion
  ...Sentry,
  // Ensure this is _not_ set
  moduleMetadataIntegration: undefined,
```

this is needed for the test to pass in ESM mode, as in ESM mode this is set on `Sentry`, which leads to the test failing!